### PR TITLE
Simplify the encoding type for default encoder

### PR DESCRIPTION
**Pull request by longquanzheng**
### Checklist
- [ ] Code compiles correctly
- [ ] Tests for the changes have been added
- [ ] All tests passing
- [ ] **This PR change is backwards-compatible**
- [ ] **This PR CONTAINS a (planned) breaking change** (it is NOT backwards-compatible)

### Related Issue
Closes #<issue_number>

### CODE REVIEW

File: src/main/java/io/iworkflow/core/JacksonJsonObjectEncoder.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public JacksonJsonObjectEncoder() {
     mapper.registerModule(new JavaTimeModule());
     mapper.registerModule(new Jdk8Module());
     mapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
-    encodingType = "BuiltinJacksonJson";
+    encodingType = "json";
```
Comment (longquanzheng) [inline src/main/java/io/iworkflow/core/JacksonJsonObjectEncoder.java:52]: The value is not being used yet. It was meant to differentiate from different encoding(e.g. compression, base64 etc). Using json is easier/shorter, and there is no much value to tell the encoding is from Spring jackson.

**Review by samuel27m — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -161,7 +161,7 @@ signing {
 }
 
 group = "io.iworkflow"
-version = "2.6.1"
+version = "2.6.2"
```
Comment (samuel27m) [inline build.gradle:164]: A future note: IMO ideally we should do version bump MRs on their own and not bundle together other code changes

Comment (longquanzheng) [reply build.gradle]: sounds good. Thanks for stopping me from being lazy :P I just separate it here: https://github.com/indeedeng/iwf-java-sdk/pull/256/files

Comment (samuel27m) [reply build.gradle]: Thanks for doing this Long 🙌

**Pull request merged**